Kickstarter-iOS/Views/Storyboards/Settings.storyboard

```diff
@@ -738,7 +738,7 @@
                                 <rect key="frame" x="0.0" y="20" width="375" height="647"/>
                                 <subviews>
                                     <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="lnb-P0-tav">
-                                        <rect key="frame" x="0.0" y="0.0" width="375" height="176"/>
```

Comment:
- Added the `SettingsFormFieldView` to the main stack view
- changed all left & right constraints within stack view elements to be relative to the `layoutMargins`, then set the `layoutMargins` of the stack view
- updated the height of all the separators to be 0.5pts (checked this value with Danny)